hooks/playbooks/compute-iscsi-config.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Increase quotas for the service project
```

Comment:
I think the task name should be changed (but let's finish the testing of the functionality first)

Reply:
LOL I failed at copy/pasting

Reply:
Done.